integration_tests/test_unsupported_types.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from robyn.robyn import Response
+
+
+def test_bad_body_types():
+    class A:
+        pass
+
+    bad_bodies = [
+        None,
+        1,
+        True,
+        A,
+        {"body": "OK"},
+        ["OK", b"OK"],
+        Response(
+            status_code=200,
+            headers={},
+            body=b"OK",
+        ),
+    ]
```

Comment:
Let's use parameterization here: https://docs.pytest.org/en/7.1.x/example/parametrize.html#different-options-for-test-ids